composer.json

```diff
@@ -14,14 +14,14 @@
         }
     ],
     "require":     {
-        "php": "^7.1",
-        "phpunit/phpunit": "^7.5",
+        "php": "^7.3 || ^8.0",
```

Comment:
Im always happy dropping old PHP versions. But it should be done for a reason other than "EOL". 
Dropping 7.1 allows us to have better type annotations, but what is the reason to drop 7.2? Will we use any features introduced in 7.3+?
I think it is a good idea to follow Symfony's example to allow PHP7.2.

Reply:
I agree with you.
Though, I would like to use this upgrade opportunity to also upgrade PHPUnit to version 9 and PHPUnit 9.0 requires PHP 7.3+ (see https://phpunit.de/announcements/phpunit-9.html)

Reply:
Yeah, that is a good reason. 

If `"phpunit/phpunit": "^8.5 || ^9.5"` will cause trouble* for us then I agree that we should drop 7.2. 

This setup we have with this library and all provider is a bit special. In a normal scenario I would prefer `symfony/phpunit-bridge`. 

\* Trouble of any kind.

Reply:
There is indeed some changes made to `assertEquals()` and `assertContains()` (optional parameters removed) in PHPUnit 9 that makes it "impossible" to use `"phpunit/phpunit": "^8.5 || ^9.5"` I think.
See https://github.com/geocoder-php/provider-integration-tests/pull/17/files#diff-d460df845bcbdd17bf0f07cd07da6ebf44e0a9437bdde656b0c1d2ecbbc5e5cb

Reply:
Okey. Thank you for investigating.